### PR TITLE
[AMDGPU] eliminate frame index v_add wave32 test

### PR DESCRIPTION
**Pull request by PankajDwivedi-25**
PR: #102346  v_add_u32_e64  test cases for wave32

### CODE REVIEW

File: llvm/test/CodeGen/AMDGPU/eliminate-frame-index-v-add-co-u32-wave32.mir [hunk 1]
```diff
@@ -0,0 +1,412 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+# Test wave32
+# RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx940 -mattr=+wavefrontsize32 -verify-machineinstrs -run-pass=prologepilog %s -o - | FileCheck -check-prefixes=FLATSCRW32,GFX940 %s
```
Comment (arsenm) [inline llvm/test/CodeGen/AMDGPU/eliminate-frame-index-v-add-co-u32-wave32.mir:3]: This run line is nonsense, wave32 is not supported for gfx9

**Review by arsenm — CHANGES_REQUESTED** [review]
Remove GFX940 run lines and checks

File: llvm/test/CodeGen/AMDGPU/eliminate-frame-index-v-add-co-u32-wave32.mir [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+# RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx1100 -mattr=+wavefrontsize32 -verify-machineinstrs -run-pass=prologepilog %s -o - | FileCheck -check-prefixes=FLATSCRW32,GFX11 %s
```
Comment (arsenm) [inline llvm/test/CodeGen/AMDGPU/eliminate-frame-index-v-add-co-u32-wave32.mir:4]: GFX11 defaults to the MUBUF ABI, this is not using FLATSCR

File: llvm/test/CodeGen/AMDGPU/eliminate-frame-index-v-add-co-u32-wave32.mir (hunk 1 above)
Comment (PankajDwivedi-25) [reply llvm/test/CodeGen/AMDGPU/eliminate-frame-index-v-add-co-u32-wave32.mir:3]: my bad will remove that.

File: llvm/test/CodeGen/AMDGPU/eliminate-frame-index-v-add-co-u32-wave32.mir (hunk 2 above)
Comment (PankajDwivedi-25) [reply llvm/test/CodeGen/AMDGPU/eliminate-frame-index-v-add-co-u32-wave32.mir:4]: should I use MUBUFW32?

Comment (arsenm) [reply llvm/test/CodeGen/AMDGPU/eliminate-frame-index-v-add-co-u32-wave32.mir:4]: yes

File: llvm/test/CodeGen/AMDGPU/eliminate-frame-index-v-add-co-u32-wave32.mir (hunk 1 above)
Comment (arsenm) [reply llvm/test/CodeGen/AMDGPU/eliminate-frame-index-v-add-co-u32-wave32.mir:3]: yes, wave32 was new in gfx10

**Review by arsenm — APPROVED** [review]

**Pull request merged**